serialization.md

```diff
@@ -167,7 +167,8 @@ Here is an example image JSON file:
             <li>amd64</li>
             <li>arm</li>
         </ul>
-        More values may be supported in the future and any of these may or may not be supported by a given container runtime implementation.
+        More values may be supported in the future and any of these may or may not be supported by a given container runtime implementation. Future values will be consistent with those already canonicalized by the <a href="https://www.debian.org/ports/">Debian ports list</a>.
```

Comment:
Indeed, for the sake of Arm, I think that it is necesary to either copy either Debian or Fedora, otherwise, we won't take into acount the specifics and there will be incompatibility problems at the kernel level. AKA, not all "ARM" images will run on all "ARM" kernels.